src/Sets/Ball1.jl

```diff
@@ -53,6 +53,8 @@ struct Ball1{N<:Real} <: AbstractCentrallySymmetricPolytope{N}
     end
 end
 
+isoperationtype(::Type{Ball1}) = false
```

Comment:
Should this not be
```suggestion
isoperationtype(::Type{<:Ball1}) = false
```
(holds for all concrete set types)
```julia
julia> typeof(rand(BallInf))
BallInf{Float64}

julia> typeof(rand(BallInf)) isa Type{BallInf}
false

julia> typeof(rand(BallInf)) isa Type{<:BallInf}
true
```

Reply:
yes, this is an omission from the concrete set types (i only added it in the operations)